terraform/ingest/index.js

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env node
+const aws = require('aws-sdk')
+const countBy = require('lodash.countby')
+const groupBy = require('lodash.groupby')
+const mapValues = require('lodash.mapvalues')
```

Comment:
A lot of Lodash equivalents have been implemented as [`Object` methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object#Methods_of_the_Object_constructor), but Node only added them in v7.  AWS runs on v6.

Other than that, Lodash seems fine.  You can require individual helpers instead of having to use the whole package, which makes the bundle a little smaller.

Reply:
Ok, gotcha!

I think if I remember right @kevinahuber was about to remove lodash when we realized lambda's on v6. lol